### PR TITLE
aws-device-farm-runner 0.0.5

### PR DESCRIPTION
**Pull request by fadookie**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

***

### 0.0.5 (2017 Jun 20)

* Incorporate patch from andreschab90: [add opportunity for metered/unmetered BillingMethod](https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/pull/26)
* [Make boolean options into drop-downs in step editor](https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/pull/29)
* [Add error message with hint about test_package_name](https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/pull/30)
* [point users to wiki for setup instructions](https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/pull/31)
* [fix up formatting of input descriptions](https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/pull/32)

### CODE REVIEW

File: steps/aws-device-farm-runner/0.0.5/step.yml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+support_url: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/issues
+published_at: 2017-06-20T18:49:18.300482063-07:00
+source:
+  git: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner.git
+  commit: 0d9e5ce0b7d7cceb6d04e2c57b0638f62d36dfe0
+host_os_tags:
+- linux
+- osx-10.9
+- osx-10.10
+type_tags:
```
Comment (viktorbenei) [inline steps/aws-device-farm-runner/0.0.5/step.yml]: should be normalized, following this guide: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention

we did normalize your previous version, you can use that as an example: https://github.com/bitrise-io/bitrise-steplib/blob/master/steps/aws-device-farm-runner/0.0.4/step.yml#L20

**Comment by viktorbenei** [comment]
@godrei & @bloodrabbit  should we allow additional tags, and just require a primary type tag to be present?

@fadookie the type tag normalization we performed a couple of months ago was to help with better step discovery in the StepLib when a user just browses through the collection. If you have any questions or feedback we're always open; for now it seems the new pre-defined type tags did work quite well e.g. in the new Workflow editor UI, but we're always open for modifications ;)

**Comment by fadookie** [comment]
@viktorbenei Thanks! I'll re-submit soon.

**Comment by viktorbenei** [comment]
Thank you @fadookie ! :)

**Comment by fadookie** [comment]
re-submitted as https://github.com/bitrise-io/bitrise-steplib/pull/1073

**Pull request closed without merging**